Neos.Fusion/Classes/FusionObjects/JoinImplementation.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Neos\Fusion\FusionObjects;
+
+/*
+ * This file is part of the Neos.Fusion package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * Fusion object to render a list of items as single concatenated string
+ */
+class JoinImplementation extends DataStructureImplementation
+{
+
+    /**
+     * Get the glue to insert between items
+     *
+     * @return string
+     */
+    public function getGlue()
+    {
+        return $this->fusionValue('__meta/glue') ?: '';
```

Comment:
Shouldn't that use the null coalescing operator instead?
```suggestion
        return $this->fusionValue('__meta/glue') ?? '';
```

Reply:
Nah, nevermind, should not make a difference in this case. (I just wanted to use the new "suggest change" feature :)